util/src/posix/time.cc

```diff
@@ -0,0 +1,14 @@
+#include <leatherman/util/time.hpp>
+
+namespace leatherman { namespace util {
+
+    void get_local_time(time_t* stored_time, tm* result){
+        localtime_r(stored_time, result);
+    }
+
+    void clear_current_line(){
+        printf("%c[2K", 27);
```

Comment:
If we do away with erasing the line, we get both cross-platform support and no printf's in one go (see my other comment on one of the above outdated diff).